1-JavaScript/Week4/homework/ex2-mondaysWorth.test.js

```diff
@@ -31,8 +34,12 @@ const mondayTasks = [
 
 const hourlyRate = 25;
 
-function computeEarnings(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function computeEarnings(mondayTasks, hourlyRate) {
+  const durationFee = mondayTasks.map(
```

Comment:
This variable seems to reference a single value while `map` returns an array.
```suggestion
  const durationFees = mondayTasks.map(
```

Reply:
I made the edit